iceprod/server/data/www_templates/job_browse.html

```diff
@@ -13,9 +13,9 @@
         <th>Job Index</th>
         <th>Status Changed</th>
         {% if (current_user) %}
-        <th><a href="javascript:call('reset_jobs');">Reset All</a></th>
-        <th><a href="javascript:call('hard_reset_jobs');">Hard Reset All</a></th>
-        <th><a href="javascript:call('suspend_jobs');">Suspend All</a></th>
+        <th><a href='javascript:set_jobs_status(task_ids_on_page, "reset", "{{ passkey }}", filter=["&task_status=failed", "&task_status=suspended"]);'>Reset All</a></th>
+        <th><a href='javascript:set_jobs_status(task_ids_on_page, "reset", "{{ passkey }}");'>Hard Reset All</a></th>
+        <th><a href='javascript:set_jobs_status(task_ids_on_page, "suspended", "{{ passkey }}");'>Suspend All</a></th>

```

Comment:
Where is `task_ids_on_page` defined?  I see `job_ids_on_page`.

Reply:
oops. Fixed in c39e108